lessons/code/cepheids.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import print_function
```

Comment:
I think this logically goes in graphics/ rather than code/, since it doesn't get run during lecture. Much of it is redundant with the simpler code they will be given in a later HW assignment, but I guess that's fine.

Reply:
Moved as advised!